index.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express')
+const {
+  getAllExotics, getExoticByName, getExoticsByType, saveNewExotic, patchExotic, deleteExotic
+} = require('./controllers/exotics')
+const {
+  getAllHunters, getHunterByTag, getHuntersBySubclass, saveNewHunter, patchHunter, deleteHunter
+} = require('./controllers/hunters')
+
+const app = express()
+
+app.use(express.static('public'))
+
+app.set('view engine', 'pug')
+
+app.get('/', (request, response) => {
+  response.redirect('/documentation')
+})
+
+app.get('/hunters', getAllHunters)
+
+app.get('/hunters/:tag', getHunterByTag)
+
+app.get('/hunters/class/:subclass', getHuntersBySubclass)
+
+app.post('/hunters', express.json(), saveNewHunter)
+
+app.patch('/hunters/:tag', express.json(), patchHunter)
+
+app.delete('/hunters/:tag', deleteHunter)
+
+app.get('/exotics', getAllExotics)
+
+app.get('/exotics/:name', getExoticByName)
+
+app.get('/exotics/:type', getExoticsByType)
```

Comment:
This is the same route. When the user types `/exotics/something` Express is matching on the `/exotics/` and placing whatever is after that (in this case `something` into the variable `name`. Express can't possibly know when the user says `something` whether that means a name or a type. It just matches the first route and puts it into the variable.